config/locales/en-GB.yml

```diff
@@ -353,12 +353,12 @@ en-GB:
   geocoder:
     search:
       title:
-        latlon: Results from <a href="http://openstreetmap.org/">Internal</a>
+        latlon: Results from <a href="http://OpenHistoricalMap.org/">Internal</a>
         ca_postcode: Results from <a href="http://geocoder.ca/">Geocoder.CA</a>
-        osm_nominatim: Results from <a href="http://nominatim.openstreetmap.org/">OpenStreetMap
+        osm_nominatim: Results from <a href="http://nominatim.OpenHistoricalMap.org/">OpenHistoricalMap
```

Comment:
@danrademacher I know this is already merged and this isn't a problem per se I don't think, but could we please have the OpenHistoricalMap in URLs be lowercase?

Reply:
Ah another reason this isn't a simple find/replace.